RoslynPluginGenerator/NuGet/NuGetPackageHandler.cs

```diff
@@ -38,6 +39,10 @@ public NuGetPackageHandler(IPackageRepository remoteRepository, string localCach
             this.logger = logger;
             this.remoteRepository = remoteRepository;
             this.localCacheRoot = localCacheRoot;
+
+            Directory.CreateDirectory(localCacheRoot);
+            this.packageManager = new PackageManager(remoteRepository, localCacheRoot);
+            this.packageManager.Logger = new NuGetLoggerAdapter(logger);
```

Comment:
Before this interface only had one method so we didn't need to keep hold of the manager instance. Now it's needed for both methods.